web/src/avatar.ts

```diff
@@ -22,16 +22,32 @@ export function build_bot_create_widget(): UploadWidget {
     const $input_error = $("#bot_avatar_file_input_error");
     const $clear_button = $("#bot_avatar_clear_button");
     const $upload_button = $("#bot_avatar_upload_button");
+    const $save_button = $("#bot_avatar_save_button");
+    const $scale_to_fit_button = $("#bot_avatar_scale_to_fit_button");
     const $preview_text = $("#add_bot_preview_text");
     const $preview_image = $("#add_bot_preview_image");
+    const $other_elements_to_hide = $(
+        '#create_bot_form .input-group label[for="bot_type"],' +
+            "#create_bot_form .input-group #create_bot_type," +
+            '#create_bot_form .input-group label[for="create_bot_name"],' +
+            "#create_bot_form .input-group #create_bot_name," +
+            '#create_bot_form .input-group label[for="bot_short_name"],' +
+            "#create_bot_form .input-group #create_bot_short_name," +
+            '#create_bot_form .input-group #avatar_upload_container span:contains("(Optional)")' +
+            ".deactivate_bot_button , .modal__footer, #optional_text, .modal-field-label, #bot-domain",
+    );
```

Comment:
This feels very messy, surely there's a better way to express your intent than a fragile list of every element.

Reply:
One theory would be to just add a shared class to these elements, but I'm not really sure what problem you're solving in the first place.

Reply:
Hey! Thanks for your review. Firstly, I would like to clarify that I worked on this feature equally with @afonsoazaruja. I mentioned him in the commit, but it doesn't seem to be showing correctly.

About the code: when trying to make Uppy’s dashboard work for the creation of a new bot, we had to hide many elements to make it look "clean." We kept adding them as there always seemed to be something missing. However, I’m sure there is a less messy way to do this, as you suggested. We will work on that.